### DISCUSSION TITLE
FBA Inbound API transition -- Will "Convert to Send to Amazon" still work in the Seller Central UI after December 20th for v0 Shipments?

### DISCUSSION THREAD

**Discussion by MetaMan2023** [amzn/selling-partner-api-models#4437] [Q&A]
@puppsupr 

A few questions on the transition date:

* Is V0 of the API being turned off ON Friday the 20th, or AFTER the 20th (ie: Midnight the 20th)

* After v0 is turned off, will "convert to send to Amazon" that occurs within Seller Central still work on shipments started with v0 before the 20th?

With v0, we currently start shipments within the API, then transition to Seller Central and click "Convert to Send to Amazon" in the SC dashboard and complete the shipments manually... so we are wondering if we start a shipment on let's say the 19th -- under normal circumstances we have a few days before we have to "Convert to Send to Amazon" (during that time we are packing and we do not complete the process until the packing is complete).

The only calls we make after that are status checks on the shipments to see what their current shipment status is.

**Comment by MetaMan2023** [comment]
BUMP

**Comment by owen109** [comment]
Would appreciate an answer to this aswell!

**Reply by MetaMan2023** [reply]
I opened this 3 days ago and asked through a case and no response on this question.   The only response I have been told is that there are no plans to extend the old v0 API.

I am holding out hope they keep the old API unofficially open for a short period as a backup in case there are issues blocking in-bounding with the new API -- but I have no basis for that.

I guess we will find out over the next few days the answer to this specific question.

**Reply by owen109** [reply]
I guess so.. thanks for letting me know. Really would be nice if amazon was more transparent about some of these changes especially when they are so impactful

**Comment by puppsupr** [comment]
Hi,
We have extended the final sunset date of Inbound v0 APIs from 12/20/24 to 1/21/25 for all SP-API developers. Here is the [announcement](https://developer-docs.amazon.com/sp-api/changelog/update-fulfillment-inbound-api-v0-removal-date-changed-to-january-21-2025) for the same.
